src/AlgebraicGeometry/Schemes/AffineAlgebraicSet/Objects/Methods.jl

```diff
@@ -66,3 +66,84 @@ function Base.show(io::IO, X::AffineAlgebraicSet)
   end
 end
 
+########################################################
+# (2) give back k-points (if finitely many)
+########################################################
+@doc raw"""
+    rational_point_coordinates(I::MPolyIdeal)
+
+Over the base field $k$, return the $k$-coordinates of the point 
+corresponding to a maximal ideal $I \in k[x_1,\dots,x_n]$, which 
+describes a $k$-point. 
+If $I$ is not maximal or does not describe a point with coordinates in the 
+field $k$, an error exception results.
+
+# Examples
+```jldoctest
+julia> R, (x, y) = QQ[:x, :y]
+(Multivariate polynomial ring in 2 variables over QQ, QQMPolyRingElem[x, y])
+
+julia> I = ideal(R, [x-1,y-3])
+Ideal generated by
+  x - 1
+  y - 3
+
+julia> rational_point_coordinates(I)
+2-element Vector{QQFieldElem}:
+ 1
+ 3
+
```

Comment:
Nice. Should we also show a non-example? E.g.
```suggestion
julia> J = ideal(R, [x-1,y^2-9])
Ideal generated by
  x - 1
  y^2 - 9

julia> rational_point_coordinates(J)
ERROR: Ideal does not describe a single K-point
```

Reply:
I deliberately tried to keep the doctest short and readable. The non-examples are tested in test/AlgebraicGeometry/Schemes/SpaceGerms.jl